tools/run_tests/run_tests.py

```diff
@@ -1383,8 +1382,8 @@ def isclose(a, b, rel_tol=1e-09, abs_tol=0.0):
     choices=[
```

Comment:
optional: Line 772: Can you update the default Linux Python test to be ran on Python 3.6 and Python 3.10?